app/main/helpers.js

```diff
@@ -12,3 +13,24 @@ export function setSavedEnv () {
   const set = makeSetSavedEnv(settings);
   return set();
 }
+
+export function readAppiumFile (argv, isPackaged, isDev) {
+  if (process.platform === 'darwin') {
+    return {success: false};
```

Comment:
why? comments would be helpful to note any platform limitations or what's going on